config.rb

```diff
@@ -17,6 +17,7 @@
   "basics/helpers.html"                  => "basics/helper_methods.html",
   "basics/getting-started.html"          => "basics/install.html",
   "basics/asset-pipeline.html"           => "advanced/asset_pipeline.html",
+  "advanced/custom-templates/index.html" => "advanced/project_templates/"
```

Comment:
I think these need to be the non-directory versions. `advanced/custom-templates.html -> advanced/project_templates.html`